src/powershell/Add-ExportScopeConfiguration.psm1

```diff
@@ -0,0 +1,137 @@
+<#
+.SYNOPSIS
+Adds an export scope configuration to the specified FinOps Hub.
+
+.PARAMETER HubName
+The name of the FinOps Hub.
+Assumes HubName = ResourceGroupName.
```

Comment:
This isn't true today. We can't guarantee this unless we change the template to be a subscription template. A few thoughts:
1. Use the `cm-resource-parent` tag, which has the original hub name.
2. Get all storage accounts that start with the safe version of the hub name.
3. Rename this to `ResourceGroupName`

1 and 2 could technically have conflicts.